### PR TITLE
update lesson with more examples of map method, including mapping use…

### PR DESCRIPTION
**Pull request by linda-seiter**
…r defined class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,40 +2,206 @@
...
 - Use the `flatMap` method
 
 ## What is Mapping?
 
-Mapping is one of the most common operations. When a collection is mapped, all
-the elements are changed to new ones according to a provided mapping function.
+- `<R> Stream<R>	map(Function<? super T,? extends R> mapper)`
+
+Mapping is one of the most common intermediate stream operations. 
+The `map` function return a new stream consisting of the results of
```
Comment (kcleland0818) [inline README.md]: Typo:  "return" should be "returns"

File: README.md [hunk 2]
```diff
@@ -2,40 +2,206 @@
...
 
+The program prints:
+
+```text
+fred
+devon
+pat
+```
+
+If we want to get the names in uppercase, we can either use a lambda
```
Comment (kcleland0818) [inline README.md]: Opinion: Possibly remove the word "either" or put the two examples (lambda vs chain calls) sequentially after the other with "Or we can chain a sequence of calls..." and then say "Both these will produce the following text" with the output.

**Review by kcleland0818 — CHANGES_REQUESTED** [review]
Looks good! Just caught one typo and opinion on just a possible reorganization (which you can choose to take or leave)

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: thanks, updated

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: thanks, updated wording

**Review by kcleland0818 — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**